src/Microsoft.ML.FastTree/FastTreeClassification.cs

```diff
@@ -275,6 +275,8 @@ protected override void InitializeTests()
         protected override BinaryPredictionTransformer<IPredictorWithFeatureWeights<float>> MakeTransformer(IPredictorWithFeatureWeights<float> model, Schema trainSchema)
         => new BinaryPredictionTransformer<IPredictorWithFeatureWeights<float>>(Host, model, trainSchema, FeatureColumn.Name);
 
+        public BinaryPredictionTransformer<IPredictorWithFeatureWeights<float>> Train(IDataView trainData, IDataView validationData = null) => TrainTransformer(trainData, validationData);
```

Comment:
>TrainTransformer [](start = 143, length = 16)

@TomFinley, @Zruty FastTree support `supportTest:true` which is not exposed in TrainTransformer method, should I extend it?

Reply:
If you're using command line, you can print out test scores in the training phase. Not sure if C# API should be consistent with that.

Reply:
Not yet. I think we are validating this for internal scenarios.

---
In reply to: [235558027](https://github.com/dotnet/machinelearning/pull/1709#discussion_r235558027) [](ancestors = 235558027)